### PR TITLE
Add some missing documentation

### PR DESCRIPTION
**Pull request by hcvdwerf**

### CODE REVIEW

File: content/docs/deployment/azure/_index.md [hunk 1]
```diff
@@ -16,6 +16,7 @@ This guide will walk you through the process of deploying an environment using A
 - **Azure CLI installed:** Make sure you have Azure CLI installed on your machine. You can install it via Homebrew with the command `brew install azure-cli`.
 - **GitHub Personal Access Token (classic):** You'll need a personal access token from GitHub with pack read permissions.
 - **Azure Account with Sufficient Permissions:** Ensure you have an Azure account with permissions that, at a minimum, allow you to create a resource group.
+- **PSQL installed:** Ensure that PSQL is installed  (eg. `sudo apt-get install postgresql postgresql-contrib -y`)
```
Comment (Markus92) [inline content/docs/deployment/azure/_index.md:19]: postgresql-contrib is not necessary and quite large

**Review by Markus92 — CHANGES_REQUESTED** [review]
Small change, other than that LGTM

File: content/docs/deployment/azure/_index.md (hunk 1 above)
Comment (hcvdwerf) [reply content/docs/deployment/azure/_index.md]: fixed

**Review by Markus92 — APPROVED** [review]
LGTM

**Comment by brunopacheco1** [comment]
@hcvdwerf thanks for you update. Can you please squash the commits and update the commit message to follow https://www.conventionalcommits.org/en/v1.0.0/?

**Comment by hcvdwerf** [comment]
> @hcvdwerf thanks for you update. Can you please squash the commits and update the commit message to follow https://www.conventionalcommits.org/en/v1.0.0/?

Squshed this two commits

**Review by brunopacheco1 — APPROVED** [review]

**Pull request merged**